specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -2932,6 +2932,102 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{operationalInsightsResourceProvider}/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/incidents/{incidentId}/alerts": {
```

Comment:
Why are these 2 new APIs POST calls instead of GETs?

Reply:
Alerts and Bookmarks are not a child resource of an incident (they can be independent resources, or nested under other resources as well). Therefore we couldn't create a GET endpoint nested under the incidents resource.

Reply:
changed the names.
the authorization mapping is already done.

Reply:
I am referring to the POST action name (the terminal URI segment).  It should be "/listAlerts", etc...

Reply:
Oh, my bad. Changing the operation ids back.
As for adding it to the URI segment, I'm not sure this is what we want. None of our other APIs have this format of "/listSomething". Please see for example the action with the operation id "Incidents_ListOfEntities". That one is also a POST action (for the same reason I mentioned above), and the format is incidents/{incidentId}/entities.

Reply:
POST calls are actions, they should be a verb.  Using a "list" prefix also allows them to be called from template deployments (although that isn't very useful in this scenario).

Reply:
I understand. However this is a POST call for more of a technical reason, rather than a logical one. The API names were a PM requirement, so we can be aligned with the rest of our calls (just incidents/{incidentId}/entities I mentioned, which is also a POST call for the same reason).